### PR TITLE
Python 3.9 Test Coverage

### PR DESCRIPTION
**Pull request by vkbo**

### CODE REVIEW

**Comment by vkbo** [comment]
A bit too quick. I thought maybe the release candidates for 3.9 were available. 3.9 is scheduled for final release on Monday 5th.

**Comment by vkbo** [comment]
Python 3.9 is now available, so this can be merged.

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -8,41 +8,13 @@ on:
 
 jobs:
   pyTestCov:
-    runs-on: ubuntu-latest
```
Comment (johtoblan) [inline .github/workflows/pytest.yml:11]: What happened to this? Included in python@v2?

Comment (johtoblan) [reply .github/workflows/pytest.yml:11]: By this, I mean lines 11-38

Comment (johtoblan) [reply .github/workflows/pytest.yml:11]: @vkbo

Comment (vkbo) [reply .github/workflows/pytest.yml:11]: I just merged the two jobs into one.

Comment (johtoblan) [reply .github/workflows/pytest.yml:11]: Oh, now I see. Thanks

Comment (vkbo) [reply .github/workflows/pytest.yml:11]: For context: They were previously two job definitions with the first having coverage, and the second being a job matrix without coverage. But I found out that codecov just merges multiple coverage reports for the same commit, so there was no point not running coverage on all jobs.

**Comment by vkbo** [comment]
You still have to approve the pull request ...

**Review by johtoblan — APPROVED** [review]

**Pull request merged**